style.css

```diff
@@ -0,0 +1,342 @@
+* {
+	margin: 0;
+	padding: 0;
+	box-sizing: border-box;
+}
+html {
+	font-family: 'Montserrat', sans-serif;
+}
+
+.container {
+	margin: 0 auto;
+	width: 100%;
+	max-width: 1150px;
+}
+
+.header {
+	width: 100%;
+	background-size: cover;
+	background-position: center;
+}
+.header_hero_bg {
+	position: relative;
+	overflow: hidden;
+	background: radial-gradient(rgb(194, 74, 126), rgb(105, 86, 235));
+}
+
+.header_nav {
+	display: flex;
+	justify-content: space-between;
+	align-items: center;
+}
+.header_nav a {
+	color: #fff;
+	text-decoration: none;
+	padding: 10px;
+	font-size: 20px;
+}
+.header_nav a:hover {
+	color: #58d5f7;
+}
+
+.header_nav .logo {
+	width: 100px;
+	height: 50px;
+	background-image: url(./images/favicon.png);
+	background-size: cover;
+	background-position: center;
+}
+
+.header-box {
+	width: 100%;
+	height: 40vh;
```

Comment:
Definiowanie wysokości na sztywno to zazwyczaj zły pomysł. Bo pojawi się więcej treści i wszystko zacznie się  rozjeżdżać (lub nie mieścić).

Lepiej jak zawartość rozpycha dany element. Wtedy też używamy `padding` do zwiększania odstępów między treścią, a daną sekcją.